internal/dlog/dlog_test.go

```diff
@@ -30,19 +31,21 @@ type params struct {
 	p, order, g *big.Int
 }
 
-func get_params() *params {
-	p, _ := emmy.GetSafePrime(20)
-	pMin1 := new(big.Int).Sub(p, big.NewInt(1))
-	x := emmy.GetRandomInt(p)
+func get_params(t *testing.T) *params {
```

Comment:
It would be better to return (*params, error) and let the caller fail the test via `t.Fatalf(...)`. This way you can also avoid passing `t` to this helper.